shared/naturalcrit/nav/nav.less

```diff
@@ -87,10 +87,15 @@ nav{
 				position         : relative;
 				display          : block;
 				width            : 100%;
-				vertical-align   : middle;
-				padding          : 8px 5px;
+				padding			 : 5px 20px 5px 12px;
 				border           : 1px solid #888;
 				border-bottom    : 0;
+				i:before {

```

Comment:
Is there a possibility of doing this on the `i` element itself instead of the `before`?

Also, if you want everything to line up with the top-level header, perhaps it makes sense to just merge this with the `.navItem i` up on line 65 so we don't have two different places that define how the icons work.